migrations/20220915172329-create-new.js

```diff
@@ -25,22 +25,25 @@ module.exports = {
           key: 'id',
         },
         onUpdate: 'CASCADE',
-        onDelete: 'SET NULL'
+        onDelete: 'SET NULL',
+      },
+      type: {
```

Comment:
Deberias haber hecho un alter no tocar la migración existente.
https://dev.to/nedsoft/add-new-fields-to-existing-sequelize-migration-3527